plugins/modules/azure_rm_aks.py

```diff
@@ -267,9 +276,19 @@
         agent_pool_profiles:
           - name: default
             count: 1
-            vm_size: Standard_DS1_v2
+            vm_size: Standard_B2s
             enable_auto_scaling: True
             type: VirtualMachineScaleSets
+            mode: System
+            max_count: 3
+            min_count: 1
+            enable_rbac: yes
+         - name: user
```

Comment:
Examples is not valid YAML.
```suggestion
          - name: user
```

Reply:
I have committed this change, thanks!